content/md/user-management/configure-saml-sso.md

```diff
@@ -0,0 +1,121 @@
+---
+id: configure-saml-sso
+themes: user-management
+title: Configure the SAML-based **Single Sign-On**
+popular: false
+ee-only: true
+related:
+priority: low
+---
+
+# Overview
+
+Single Sign-On (aka SSO) is an authentication process that allows a user to access multiple applications with only one set of credentials.
+
+Single Sign-On is a common practice around corporations that handles lots of users and lots of applications.
+
+## Common SAML terms
+
+Understanding how SAML-based Single Sign-On work can be a complicated task as it introduces terms that can be obscure at first glance.
+
+### Identity Provider (IdP)
+
+The `Identity Provider` is the authority that verifies the user's identity and grant access to a requested application (aka the `Service Provider`)
+
+### Service Provider (SP)
+
+The `Service Provider` is the hosted resource that the user tries to access. In our scenario, the SP is a part of the PIM that handles the authentication process.
+
+### Entity ID
+
+An Entity ID is a globally unique name for an `Identity Provider` or a `Service Provider`. This unique name is used to identify each parties in the SSO process.
+For the `Service Provider`, the Entity ID is automatically generated and corresponds by default to the metadata URL of the SP.
+
+### Assertion Consumer Service (ACS)
+
+The ACS is the SP endpoint (URL) that is responsible for receiving the SAML response from the IdP.
+
+### Metadata
+
+Metadata is a set of information supplied either by the IdP or the SP, in XML format.
+* `Service Provider`: The SP supplied metadata provide the Entity ID, the X.509 certificate used for decrypting messages coming from the SP, the Logout URL and the ACS URL.
+
+:::tips
+The SP metadata information can be accessed at the following URL: https://YOUR_PIM_SERVER/saml/metadata. Depending on your IdP, this metadata URL can be imported into the IdP to ease the configuration process of your application.
+:::
+
+* `Identity Provider`: The IdP can also supply metadata to help configure the SAML communication. At this time, it is not possible to import it in the PIM to configure the Single Sign-On.
+
+## What is SAML?
+
+SAML (Security Assertion Markup Language) is an XML-based standard for exchanging authentication data between an `Identity Provider (IdP)` (such as Microsoft Azure AD) and a `Service Provider (SP)` (in our case our beloved PIM).
+
+# Gather information from your Identity Provider (IdP)
```

Comment:
It's a bit weird to have a section only for this. It feels empty. As this is the first step of the set up, maybe put it into the next section as subsections.